code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm

```diff
@@ -841,3 +838,22 @@
 			C.anchored = 0
 	else
 		return
+
+/mob/living/carbon/human/proc/hastemode()
+	set name = "Haste Mode"
+	set desc = "Run slightly faster with high nutriment drain."
+	set category = "Abilities"
+
+	var/mob/living/carbon/human/C = src
+	if(C.haste_run != 1)
+		C.haste_run = 1
+		to_chat(C, "<span class='notice'>Haste Mode enabled.</span>")
+		C.species.slowdown -= 0.2
```

Comment:
wait, if I were to use this as a xenochimera, would this not affect _every chimera in the game?_ You're changing the _species_ values.

Reply:
Hm yeah that might be quite an oversight I forgot to think about.

Reply:
You could achieve the effect by putting an "if (haste_run) tally -= 0.2" in movement_delay() somewhere after this line:

https://github.com/VOREStation/VOREStation/blob/1aa1843fff22ebba745a24cd3a8480746e79bde5/code/modules/mob/living/carbon/human/human_movement.dm#L7

That wouldn't affect metabolism or any of the other species things though.